site/constants/index.ts

```diff
@@ -31,10 +33,17 @@ export const DISRUPTION_REASONS: {
     },
 ];
 
+export const contactFeedbackQuestion = "Did you contact us for assistance at any point?";
```

Comment:
I'd stick to the capital format here `CONTACT_FEEDBACK_QUESTION` etc.

Reply:
done